Snippets/Core/Core_6/UpgradeGuides/5to6/UpgradeMessageHandler.cs

```diff
@@ -37,4 +37,88 @@ await context.Publish(new MyEvent())
         }
     }
     #endregion
+
+
+    #region 5to6-handler-migration-beginning
+    public class MigrationBeginning : IHandleMessagesFromPreviousVersions<MyMessage>
+    {
+        public IBus Bus { get; set; }
+
+        public void Handle(MyMessage message)
+        {
+            Bus.Send(new MyOtherMessage());
+            Bus.Publish(new MyEvent());
+        }
+    }
+    #endregion
+
+    #region 5to6-handler-migration-step1
```

Comment:
Thought about it as well but I think it is crucial to see the whole picture

On Tuesday, June 28, 2016, Indu Alagarsamy <notifications@github.com> wrote:

> In Snippets/Core/Core_6/UpgradeGuides/5to6/UpgradeMessageHandler.cs
> <https://github.com/Particular/docs.particular.net/pull/1642#discussion_r68813729>
> :
>
> > +
> > +
> > +    #region 5to6-handler-migration-beginning
> > +    public class MigrationBeginning : IHandleMessagesFromPreviousVersions<MyMessage>
> > +    {
> > +        public IBus Bus { get; set; }
> > +
> > +        public void Handle(MyMessage message)
> > +        {
> > +            Bus.Send(new MyOtherMessage());
> > +            Bus.Publish(new MyEvent());
> > +        }
> > +    }
> > +    #endregion
> > +
> > +    #region 5to6-handler-migration-step1
>
> Since the previous snippet shows the entire class, do we need to repeat
> the entire code again? Why not just have the new Handle method enclosed in
> a region? I think that'd be much better to read.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/docs.particular.net/pull/1642/files/cb3b63006af3e1ed67464dfd0f4af1820aa83cc0#r68813729>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAKosho3XhYTtD9rkbppiUqGPNjebg0oks5qQWaggaJpZM4JACq2>
> .
>